Marlin/language_gl.h

```diff
@@ -182,6 +185,8 @@
 #define MSG_INFO_EXTRUDERS                  "Extrusores"
 #define MSG_INFO_BAUDRATE                   "Baudios"
 #define MSG_INFO_PROTOCOL                   "Protocolo"
+#define MSG_LIGHTS_ON                       "Acender luz LCD"
+#define MSG_LIGHTS_OFF                      "Apagar luz LCD"
```

Comment:
@Blue-Marlin You are right: I have reviewed in code, those strings are in ultralcd.cpp lcd_main_menu() and I got confused. 
@thinkyhead I new PR is comming.
Thank you 👍